extensions/stackdriver/config/stackdriver_plugin_config.proto

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package stackdriver.config;
+
+message PluginConfig {
+  // Indicates whether the module should act as inbound or outbound reporter.
+  // Difference being:
+  //   * inbound and outbound have different metric types to export.
+  //   * inbound reporter writes access logs.
```

Comment:
Is this still true given there is a boolean `export_access_log` below?

Reply:
Yes, I think we just want to export access log at server side. `export_access_log` controls if server exports access log or not.